packages/gestalt/src/Spinner/VRSpinner.css

```diff
@@ -0,0 +1,111 @@
+.spinner {
+  --g-dot-size: calc(var(--g-size) / 3);
+  --g-dot-space: calc(var(--g-dot-size) * 1.75);
+
+  height: var(--g-size);
+  width: var(--g-size);
+}
+
+.lg {
+  --g-size: var(--comp-spinner-size-lg);
+}
+
+.md {
+  --g-size: var(--comp-spinner-size-md);
+}
+
+.sm {
+  --g-size: var(--comp-spinner-size-sm);
+}
+
+.spinnerFrame {
+  animation: spin 1500ms linear infinite;
+  box-sizing: border-box;
+  height: calc(var(--g-dot-space) * sqrt(3) / 2);
+  left: calc(var(--g-dot-size) * 0.625);
+  position: relative;
+  top: calc(var(--g-dot-size) / 2);
+  transform-origin: center calc(var(--g-dot-space) / sqrt(3));
+  width: var(--g-dot-space);
+}
+
+.spinnerFrame > div {
+  animation-composition: add;
+  animation-duration: 800ms, 1.8s;
+  animation-iteration-count: infinite;
+  animation-name: scale, colors;
+  animation-timing-function: linear;
+  border-radius: 50%;
```

Comment:
can we tokenize this? is this equal to circle?

Reply:
i dont understand why we should tokenize the dots of spinner. these rules are specific to the component and private.

Reply:
We could tokenize all the calculations in the file too. but i see no point. we could absolutely tokenize every single value here, but we don't do it, right?